tests/testthat/test-model.R

```diff
@@ -165,17 +168,24 @@ if (NOT_CRAN) {
     stepsize = 0,
     adapt_engaged = "NO",
     adapt_delta = 2,
-    save_diagnostics = "NOT_LOGICAL"
+    save_diagnostics = "NOT_LOGICAL",
+    init_buffer = "NOT_INTEGER",
+    term_buffer = "NOT_INTEGER",
+    window = "NOT_INTEGER"
   )
 
   bad_arg_values_2 <- list(
     init = "NOT_A_FILE",
     seed = 1:10,
     stepsize = 1:10,
-    metric = c("AA", "BB")
+    metric = c("AA", "BB"),
+    init_buffer = -5,
+    term_buffer = -6,
+    window = -7
   )
 
   bad_arg_values_3 <- list(
+    data_file = rep("NOT_A_FILE", 5),
```

Comment:
There actually isn’t a `data_file` argument to the sample method. All options are handled via `data` and the name`data_file` is just used internally. So this would error but because the argument doesn’t exist. 

I think the best option might be to change `data_file` to `data` here and then assert that the internal `self$data_file` is length 1 when validating the CmdStanArgs object. I’m presuming we want it to error if a user tries to specify multiple data files, right?

Reply:
I just removed this test. I guess we'd want to error if we had a list of lists or something?

But there wasn't a bug I was fixing with this -- just saw the tests were unused and needed em' so I put them back. So I just removed the leftover test.